scripts/xpost.coffee

```diff
@@ -12,13 +12,15 @@
 #   No notes needed
 #
 # Author:
-#   @afeld with ideas from @wslack
+#   @afeld and @wslack
 
 module.exports = (robot) ->
   console.log("XPOST script loaded.")
   robot.hear /\bxpost #([\w\-]+)/i, (msg) ->
-    channel = msg.match[1]
-    console.log("cross-posting to ##{channel}")
-
+    target = msg.match[1]
+    poster = msg.message.user.name
     text = msg.message.text
-    robot.messageRoom(channel, text)
+#    console.log("cross-posting to #{target}")
```

Comment:
I prefer the practice of deleting these lines altogether before merging. They're easy enough to add back in if you need for testing further updates.